### DISCUSSION TITLE
[Bug]: Mobile APIs not compatible

### DISCUSSION THREAD

**Discussion by manuelweisser** [RafaelGB/obsidian-db-folder#699] [General]
### What happened?

I cannot install the Plugin on mobile versions (iPad & Android phone). 

Normal instal via the Obsidian Plugin manager returns „No appropriate version found“. 

Trying to install the beta version using BRAT, it says that the required Obsidian is 1.1.1, but available is 1.0.3. 
Note that I have both Obsidian apps updated to the newest version, which is „1.4.1. (83)“, specifying the „API v1.0.3“. 

Do I simply need to wait for the apps to follow some internal API update?

### What platform were you using?

Mobile

### Version of the plugin

3.1.0

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Sorry for the inconvenience. Obsidian change the edition API and it is necessary for the last version.

Once the version is released on mobile it will be avaliable again.

While its ready, you can install old versions using BRAT plugin (<=3.0.1 run cool on mobile)

**Reply by ndreux** [reply]
Hey @RafaelGB 👋

I have the same issues as mentioned above, even while targeting 3.0.1 with BRAT.

`Trying to install the beta version using BRAT, it says that the required Obsidian is 1.1.1, but available is 1.0.3.`

**Comment by RafaelGB** [comment]
Commit 21aeb83c

I will change it temporary, sure

**Reply by ndreux** [reply]
I could proceed with the installation 👌

Thanks @RafaelGB !

**Comment by lilongcnc** [comment]
您好，你的邮件我已收到。我尽快处理的哈~~么么哒~~~

**Comment by RafaelGB** [comment]
Commit 012f9624


Added a temporal solution about. Try mobile with 3.2.3
